build.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
     implementation "com.esri.arcgisruntime:arcgis-java:$arcgisVersion"
     natives "com.esri.arcgisruntime:arcgis-java-jnilibs:$arcgisVersion"
     natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
+    runtimeOnly "org.slf4j:slf4j-nop:1.7.32"
```

Comment:
Samples have this comment above the slf4j line:
`// handle SLF4J http://www.slf4j.org/codes.html#StaticLoggerBinder`